OpenOversight/app/main/views.py

```diff
@@ -690,6 +689,7 @@ def add_department():
                 short_name=form.short_name.data,
                 state=form.state.data,
                 created_by=current_user.get_id(),
+                last_updated_by=current_user.get_id(),
```

Comment:
Since departments are not going through the Mixin-derived creation method, I made sure to add both fields.